src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
That's the only correct URL for the legend produced by the ArcGIS Server; I agree with that.  But you could point NM to a separate, hand-generated legend URL.  We used to do this with the Broadband layers.